keps/sig-storage/20190122-csi-inline-volumes.md

```diff
@@ -241,8 +256,11 @@ type CSIVolumeSource struct {
 ### Secret references
 Secret references declared in an inline CSI volume can only be used with namespaces from pods where they are referenced .  For inline usage, secret references are stored in `LocalObjectReference` values:
 * `LocalObjectReference` do not include a namespace reference.  This is to prevent reference to arbitrary namespace values.
-* The namespace reference will be extracted from the the pod spec at different phases of storage lifecycle by the Kubelet or external CSI compoennts.
+* The namespace reference will be extracted from the the pod spec at different phases of storage lifecycle by the Kubelet or external CSI components.
 * The Kubelet and external CSI components must ensure secret references can only be used with the namespace from inline pod spec.
+* Persistent inline volumes will not support secrets during attach/detach operations (see above for detail)
```

Comment:
we should limit the secret fields to ones we intend to support (drop ControllerPublishSecretRef) in the inline CSIVolumeSource typedef above

Reply:
Yes. Agreed!